pkg/lisp/rootfs/usr/bin/sudo

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
```

Comment:
This is a bit off. `sudo` as a script that just `exec`s its args? Why not just install `sudo` via `apk add sudo`, either in the `pkg/lisp/Dockerfile` or in the `pkg/zedctr/Dockerfile`?

Reply:
Good catch! There were some issues with LISP using it that required us to put this ugly hack in, but I believe those are gone -- so I'll remove the hack (I knew these PR reviews were good for something! ;-))